packages/i/inja/xmake.lua

```diff
@@ -15,6 +16,10 @@ package("inja")
     end)
 
     on_test(function (package)
+        local cxx_std = "c++11"
+        if package:version() == nil or package:version():satisfies(">=3.4.0") then
+            cxx_std = "c++17"
+        end
```

Comment:
```
local cxx_std = "c++17"
if package:version() and package:version():le("3.4.0") then
        cxx_std = "c++11"
end
```

Reply:
cool , the development documentation should explain the usage of semver versioning